README.md

```diff
@@ -167,21 +167,11 @@ We have administration view [Leto-Modelizer-Admin](https://github.com/ditrit/let
 
 ```json
 {
-    "backend": {
-        "url": "http://localhost:1337",
-        "appId": "leto-modelizer-api-dev",
-        "adminUrl": "http://localhost:9000"
-    }
+  "backendUrl": "https://localhost:8443"
 }
 ```
 
-To get authentication setup, `backend.url` and `backend.appId` are mandatory.
-To get administration view, all the fields inside `backend` are mandatory.
-
-Here's a description of each key in the provided configuration:
-- `url`: the url of the backend.
-- `appId`: the application ID of the backend.
-- `adminUrl`: the url of the administration application.
+To get the authentication setup `backendUrl`is mandatory.
 
 **_NOTE:_**: If the previous configuration is not present in the configuration file, Leto-Modelizer will be launched with the backend mode deactivated.
 **_NOTE:_**: For now, there is no UI associated to the backend, but the UI for the admin is coming soon !
```

Comment:
Can you fix the double `:` for the two `**_NOTE:_**: please ?

Reply:
Also, check lines 252-255, the global config file is mentionned again so must be updated

Reply:
done